pkg/api/query/v1.go

```diff
@@ -418,7 +418,7 @@ func analyzeQueryOutput(query promql.Query) (queryTelemetry, error) {
 	if eq, ok := query.(engine.ExplainableQuery); ok {
 		return processAnalysis(eq.Analyze()), nil
 	}
-	return queryTelemetry{}, errors.Errorf("Query not analyzable; change engine to 'thanos'")
+	return queryTelemetry{}, errors.Errorf("Query fallback to prometheus engine; not analyzable. ")
```

Comment:
What if they use Prometheus engine? Then it is not fallback. This can be done through the API.